theme/errorList.less

```diff
@@ -1,48 +1,60 @@
 @summary-hover-color: #00000067;
 
+// Expanded error list
 .errorList {
     position: relative;
-    min-height: 200px;
+    max-height: 16em;
+    min-height: 16em;
+    border-top: 2px #e5e5e5 solid;
 
-    .errorListInner {
-        height: 200px;
-        border-top: 1px #e5e5e5 solid;
+    .errorListHeader {
         padding: 1em;
-        overflow-y: scroll;
+        height: 4em;
+        margin-bottom: 0;
 
         h4 {
-            margin-bottom: 0.5em;
+            margin: 0;
+            display: inline-block;
+            vertical-align: middle;
+        }
+
+        .countBubble {
+            margin-left: 1em;
         }
 
-        .closeIcon {
+        .collapseButton {
             float: right;
             cursor: pointer;
-            position: absolute;
-            top: 1em;
-            right: 1em;
+            position: relative;
+            vertical-align: middle;
 
             i {
-                font-size: 2em;
+                font-size: 1.5em;
             }
 
             i:not(:hover) {
                 opacity: 0.5;
             }
         }
     }
+
+    .errorListInner {
+        height: 12em;
```

Comment:
Since this height is 16em - 4em from the header, it's probably worth using LESS variables and math here.
https://stackoverflow.com/questions/17433849/less-subtract-from-variable